flink-tests/src/test/java/org/apache/flink/test/checkpointing/IgnoreInFlightDataITCase.java

```diff
@@ -68,13 +70,15 @@
                             .build());
 
     @Rule public final SharedObjects sharedObjects = SharedObjects.create();
+    @Rule public Timeout timeout = new Timeout(30, TimeUnit.SECONDS);
```

Comment:
Since some time we discourage timeouts in JUnit tests. Please see `Avoid timeouts in JUnit tests` here https://flink.apache.org/contributing/code-style-and-quality-common.html#design-for-testability

Reply:
Ok, I have removed commit with timeotu